BeerProject/src/main/java/wawer/kamil/beerproject/service/UserService.java

```diff
@@ -0,0 +1,23 @@
+package wawer.kamil.beerproject.service;
+
+import org.springframework.data.domain.Page;
+import org.springframework.data.domain.Pageable;
+import wawer.kamil.beerproject.model.User;
+import wawer.kamil.beerproject.exceptions.NoContentException;
+
+import java.util.List;
+
+public interface UserService {
```

Comment:
Jeżeli twoje wyjątki będą rozszerzeniem klasy `RuntimeException` a nie `Exception` to nie będziesz musiał ich rzucać wyżej a i tak będą obsługiwane w Advicerze - kod będzie zdecydowanie czytelniejszy